usvg/src/writer.rs

```diff
@@ -25,15 +25,34 @@ impl<T: Default + PartialEq + Copy> IsDefault for T {
 }
 
 /// XML writing options.
-#[derive(Clone, Default, Debug)]
+#[derive(Clone, Debug)]
 pub struct XmlOptions {
     /// Used to add a custom prefix to each element ID during writing.
     pub id_prefix: Option<String>,
 
+    /// Set the coordinates numeric precision.
```

Comment:
Add a new line after the first sentence, so docs would be rendered correctly.

And add a period at the end of the second sentence.

Reply:
I always forget about that